src/ZprimeSelectionModule.cxx

```diff
@@ -141,7 +144,13 @@ bool ZprimeSelectionModule::process(Event & event){
   jetlepton_cleaner->process(event);
   jetER_smearer->process(event);
   jet_cleaner1->process(event); // jets w/ pt>25 GeV for lepton-2Dcut
-  bool pass_twodcut = twodcut_sel->passes(event); // lepton 2D-cut
+  //Trigger selection
+  bool ele_Trigger_selection = ele_Trigger->passes(event);
+  bool mu_Trigger_selection = mu_Trigger->passes(event);
+  if (!ele_Trigger_selection || !mu_Trigger_selection || (mu_Trigger_selection && ele_Trigger_selection)) return false;
```

Comment:
yeah, that s basically what i meant. "loading" was the wrong verb, apologies (actually that never happens because only one of "ele_Trigger" or "mu_Trigger" is actually loaded, the other is just standing there). the PR is closed anyway, but, if it's ok with Daniel, he should probably follow up on your suggestion and update it.
maybe, i also got your point on hard-coding the HLT path, the risk would be basically that one runs, say, the muon selection leaving the string of the electron HLT path. if that's your point, i get it now (but if one does that, that's on him ;))